ruby/inc/leatherman/ruby/api.hpp

```diff
@@ -266,6 +266,10 @@ namespace leatherman {  namespace ruby {
         /**
          * See MRI documentation.
          */
+        VALUE (* const rb_ll2inum)(SIGNED_VALUE);
```

Comment:
oh, whoops. Copy-paste error. let me check what the actual signature of that function is